content/Interrupts.md

```diff
@@ -1,27 +1,27 @@
 ### IME - Interrupt Master Enable Flag (Write Only)
 
 ```
-0 - Disable all Interrupts
-1 - Enable all Interrupts that are enabled in IE Register (FFFF)
+0 - Disable all interrupts
+1 - Enable all interrupts that are enabled in the IE register (FFFF)
 ```
 
 The IME flag is used to disable all interrupts, overriding any enabled
-bits in the IE Register. It isn't possible to access the IME flag by
-using a I/O address, instead IME is manipulated by
-the following instructions/operations:
+bits in the IE register. It isn't possible to access the IME flag by
+using a I/O address. IME can be modified by
+the following instructions/operations only:
 
 ```
-EI     ;Enable Interrupts  (that is, IME=1)
-DI     ;Disable Interrupts (that is, IME=0)
-RETI   ;Enable Ints & Return (same as the instructions combination EI, RET)
-<INT>  ;Disable Ints & Call to Interrupt Vector
+EI     ;Enable interrupts  (that is, IME=1)
```

Comment:
I present tense should be used instead of imperative, so the `<INT>` entry makes more sense.

Reply:
Not sure if this is what you wanted but I have tried to fix it.

Reply:
I meant e.g. "Enables interrupts and returns"